### PR TITLE
fix the arch migrator

### PR DESCRIPTION
**Pull request by jayfurmanek**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

closes #48 

I updated the pin for cppzmq to 4.4.1 as that is the version available for the new arches. It builds and passes the tests ok, but will need a review.
Perhaps we revisit the need for that pin in the recipe?

### CODE REVIEW

**Comment by jayfurmanek** [comment]
@conda-forge-admin, please rerender

**Comment by JohanMabille** [comment]
The tests are only to check that the build generated all required files. There is no runtime tests (maybe we should add them to the recipe at some point).

I will build and test xeus with cpzmq 4.4.1 as soon as I can.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,21 +12,22 @@ source:
...
     - cmake
+    - make  # [unix]
     - pkg-config  # [unix]
     - {{ compiler('cxx') }}
   host:
     - xtl >=0.6.1,<0.7
     - nlohmann_json >=3.2.0,<4.0
     - libuuid  # [linux]
-    - cppzmq 4.3.0
+    - cppzmq 4.4.1
```
Comment (JohanMabille) [inline recipe/meta.yaml]: We can lessen the constraint here and use `cppzmq >= 4.3.0, <5

**Comment by JohanMabille** [comment]
Thanks!

**Comment by jayfurmanek** [comment]
Thanks @JohanMabille . I notice the ppc64le build was published, but the aarch64 one was not. Do you have privileges to check on why?
https://anaconda.org/conda-forge/xeus/files

**Comment by JohanMabille** [comment]
It looks like there is no build for aarch64 in the CI. This PR (like the original one opened by the bot) adds a travis configuration for ppc64Ie only. There's a `drone.yml` file containing the steps for a aarch64 build, but it is not included anywhere.

I'm gonna ask on [gitter](https://gitter.im/conda-forge/conda-forge.github.io)

**Pull request merged**